crates/sui-framework/packages/sui-framework/sources/kiosk/kiosk.move

```diff
@@ -385,7 +392,7 @@ module sui::kiosk {
     }
 
     /// Check whether the `KioskOwnerCap` matches the `Kiosk`.
-    public fun has_access(self: &mut Kiosk, cap: &KioskOwnerCap): bool {
+    public fun has_access(self: &Kiosk, cap: &KioskOwnerCap): bool {
```

Comment:
@damirka and I chatted about this offline, but just to catch up the PR thread -- this won't work unfortunately, it counts as a signature change on a public function.  Happily the framework compatibility test catches this too, so good to know that our CI is working here!